src/data/markdown/docs/02 javascript api/07 k6-experimental/01 browser.md

```diff
@@ -9,55 +9,106 @@ The browser module APIs aim for rough compatibility with the [Playwright API for
 
 Note that because k6 does not run in NodeJS, the browser module APIs will slightly differ from their Playwright counterparts.
 
+You can find examples of using [the browser module API](#browser-module-api) in our [getting started guide](/using-k6-browser).
+
 <Blockquote mod="note" title="">
 
-To work with the browser module, make sure you are using [k6 version 0.43.0](https://github.com/grafana/k6/releases/tag/v0.43.0) or above.
+To work with the browser module, make sure you are using the latest [k6 version](https://github.com/grafana/k6/releases).
 
 </Blockquote>
 
-## Modules
+## Importable Properties
```

Comment:
Is saying "Importable Properties" more accurate than saying "Modules"? Could we say "Properties", and just say "properties you can import" in the sentence below?

Reply:
They are not modules but the properties of the browser module. I'm changing the sentence to this:

> The table below lists the properties you can import from the top-level module (`'k6/experimental/browser'`).

Reply:
Please see: a4beb463.